internal/util-complete/src/main/scala/sbt/internal/util/LineReader.scala

```diff
@@ -120,7 +120,10 @@ private[sbt] object JLine {
   def usingTerminal[T](f: jline.Terminal => T): T =
     withTerminal { t =>
       t.restore
-      f(t)
+      val result = f(t)
+      // UnixTerminal::init call removes echo
+      t.setEchoEnabled(true)
```

Comment:
Could you try `t.restore` here instead?
I am guessing that would bring the state back to whatever the terminal was before.

Reply:
Thx @eed3si9n. Hope to check that today afternoon. Will give feedback here. Best regards :)